app/controllers/application_controller.rb

```diff
@@ -48,7 +49,8 @@ def after_sign_in_path_for(resource)
     return edit_user_path id: current_user.id if session[:pending_organisation_id]
     return organisation_path(current_user.organisation) if current_user.organisation
     return session[:previous_url] if session[:previous_url]
-    return organisation_path(Organisation.find(current_user.pending_organisation_id)) if current_user.pending_organisation_id
+    return organisation_path(Organisation.find(current_user.pending_organisation_id)) if
```

Comment:
right, but we can still use the approach I suggest right? Since we now we have a separate method to pull that into